backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -53,15 +53,20 @@ def get_collections_list(from_date: int = None, to_date: int = None, user: Optio
 @dbconnect
 def get_collection_details(collection_uuid: str, visibility: str, user: str):
     db_session = g.db_session
-    collection = Collection.get_collection(db_session, collection_uuid, visibility)
+    collection = Collection.get_collection(db_session, collection_uuid, visibility, include_tombstones=True)
     if not collection:
         raise ForbiddenHTTPException()
-    get_tombstone_datasets = (
-        _is_user_owner_or_allowed(user, collection.owner) and collection.visibility == CollectionVisibility.PRIVATE
-    )
-    result = collection.reshape_for_api(get_tombstone_datasets)
+    if collection.tombstone == True and visibility == CollectionVisibility.PUBLIC.name:
+        result = dict(contact_name=collection.contact_name, contact_email=collection.contact_email)
```

Comment:
Brian got back to me on this, we're going to return just the tombstone with no data attached.  A bit naive, but safe for now until we revisit later with the dataset tombstone response as well

Reply:
Can you expand on "just the tombstone with no data". What would that response look like?

Reply:
Actually I guess that would just be empty contents and just the 410